data/lib/src/network/remote_exception_thrower.dart

```diff
@@ -53,8 +53,14 @@ class RemoteExceptionThrower {
             throw InternalServerError();
           }
           break;
-        default:
```

Comment:
IMO, delete fallback case is not a good idea. We can not know what will happen to us.